### PR TITLE
Create hierarchy

### PR DESCRIPTION
**Pull request by Sameerthe-Big-O**

### CODE REVIEW

**Comment by Sameerthe-Big-O** [comment]
i'll add the code of each functionality step by step so that's why i've removed the whole previous code

File: routes/employee/controller.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const controller = {
 
     add: async (req, res) => {
 
-        const addResponse = await EmployeeServices.add(req.body, req.file.path);
+        const addResponse = await EmployeeServices.adsd(req.body, req.file.path);
```
Comment (AbubakarZorrain) [inline routes/employee/controller.js:24]: What is adsd? Rename it correctly.

File: services/hierarchy.js [hunk 2]
```diff
@@ -1,56 +1,16 @@
...
-        return { message: "success", data: savedData };
-      }
-    } catch (error) {
-      return { message: "error", data: error.message };
-    }
-  },
-};
-
-export default hierarchySerive;
+export default hierarchySerive;
```
Comment (AbubakarZorrain) [inline services/hierarchy.js:16]: What is hierarchySerive, rename it correctly.

File: validations/hierarchy.validation.js [hunk 3]
```diff
@@ -30,4 +30,4 @@ const validations ={
  } 
 }
 
-export default validations;
\ No newline at end of file
+export default validations
```
Comment (AbubakarZorrain) [inline validations/hierarchy.validation.js:33]: Rename it as per file name.

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

File: config/env.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 const environment = {
   port: process.env.PORT || 2022,
   nodeEnv: process.env.NODE_ENV || "development",
-  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/boilerplate",
+  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/HRMS",
```
[the remark this answers is not in the document]
Comment (Sameerthe-Big-O) [reply config/env.js:4]: sir i just tried using MongoDB live but i was having some error and time is runing so just for now i'll be working with local if you approve

**Pull request closed without merging**